app/browsered/field-validation-checks.js

```diff
@@ -116,10 +117,19 @@ exports.isNotVatNumber = value => {
 
 exports.isNotCompanyNumber = value => {
   const sanitisedCompanyNumber = value.replace(/\s/g, '').toUpperCase()
+
+  // check explicitly for empty string and make it valid,
+  // to be able to handle the validation on the client side.
+  // the empty string validation will be checked on server side.
```

Comment:
maybe a bit clearer:

Allow an empty string, as company number is not always required. If it is required, this will be flagged by the server side validation.

Reply:
Yes, thanks, will change it in the next PR